runtime/syscall-server/syscall_context.cpp

```diff
@@ -46,6 +50,37 @@
 			    .imm = IMM })
 #endif
 
+#if __linux__ && !BPFTIME_BUILD_WITH_LIBBPF
+#define offsetofend(type, member) (offsetof(type, member) + sizeof(((type *)0)->member))
+static void *libc_handle = dlopen(LIBC_SO, RTLD_LAZY);
+static auto libc_syscall =
```

Comment:
Original syscall from libc was already stored at https://github.com/eunomia-bpf/bpftime/blob/b0ac6d9a1204b2c94da43132b70d22024a445f9d/runtime/syscall-server/syscall_context.hpp#L92

Reply:
above definition is to support the `bpf_get_obj_by_fd` function. I see that there is a place where original syscall is being stored, but it is defined inside a member function. Do you want me to replace `libc_handle` with `syscall_fun` ?

Reply:
Where was bpf_get_obj_by_fd called?

Reply:
wrong function at my end. It was actually `bpf_obj_get_info_by_fd`

Reply:
> bpf_obj_get_info_by_fd

`bpf_obj_get_info_by_fd` is only called by member function of syscall_context, so it's ok to use functions stored in syscall_context